resource/src/main/java/za/co/sfy/lucid/sams/resource/SubjectsResource.java

```diff
@@ -23,7 +24,85 @@ public SubjectsResource(EdusolStrucDatabaseConnectionManager edusolStrucDatabase
 
     @Override
     public PreparedStatement save(Connection connection, Object object) throws LucidSamsExecutionException {
-        return null;
+       
+        String sql = "INSERT INTO " + TABLE_NAME + "(Name, Key, Code, Group, Selected, Phase, Afrname, PrintOrder, "
+                        + "CTAWeight, ExcludeSchedule, LuritsCode, SubjectStatus, SubjectGrade, OfficialSubjectID)"
+                        + "VALUES(?,?,?,?,?,?,?,?,?,?,?,?,?,?)";
+
+        try { 
+            PreparedStatement preparedStatement = connection.prepareStatement(sql);
+            Subjects subjects = (Subjects) object;
+            preparedStatement.setString(1, subjects.getName());
+            preparedStatement.setString(2, subjects.getKey());
+            preparedStatement.setString(3, subjects.getCode());
+            preparedStatement.setString(4, subjects.getGroup());
+
+             Integer Select = subjects.getSelected();
+            if (null == Select) {
+                preparedStatement.setNull(5, Types.INTEGER);
+            } else {
+                preparedStatement.setInt(5, Select);
+            }
+
+              Integer Phase = subjects.getPhase();
+            if (null == Phase) {
+                preparedStatement.setNull(6, Types.INTEGER);
+            } else {
+                preparedStatement.setInt(6, Phase);
+            }
+
+            preparedStatement.setString(7, subjects.getAfrname());
+
+              Integer PrintOrder = subjects.getPrintOrder();
+            if (null == PrintOrder) {
+                preparedStatement.setNull(8, Types.INTEGER);
+            } else {
+                preparedStatement.setInt(8, Select);
+            }
+
+              Integer CTAWeight = subjects.getCTAWeight();
```

Comment:
naming convention violation, please read up on java naming conventions and apply changes everywhere where necessary.